lib/ansible/plugins/lookup/skydive.py

```diff
@@ -39,12 +39,14 @@
 EXAMPLES = r"""
 - name: return skydive metdata if present based on Name
   set_fact:
-    skydive_meta: "{{ lookup('skydive', filter={'query': \"G.V().Has('Name', 'sumit-VirtualBox')\"}) }}"
+    skydive_meta: |
```

Comment:
The original was correct Ansible code, but unfortunately not something YAML+Jinja2 can parse. The problem is that Ansible first loads the file as YAML, and then interprets strings in there as Jinja2. This means that Jinja2 will get the string with `\"` converted to `"`. In the YAML+Jinja2 lexer, the Jinja2 lexer gets everything between `{{` and `}}`, which due to `\"` is invalid Jinja2.

I've rewritten it so that this is both valid Ansible code and valid YAML+Jinja2.

Reply:
Right, I personally use and prefer:
```suggestion
    skydive_meta: >-
```
(which also trims spaces)

Reply:
That's even better! Thanks!